src/elements/Button/Buttons.js

```diff
@@ -12,6 +12,7 @@ export default class Buttons extends Component {
     library: META.library.semanticUI,
     name: 'Buttons',
     type: META.type.element,
+    parent: 'Button'
```

Comment:
Buttons is not a top level component, but a child to the Button component.   This prevents it from showing in the menu.